src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -298,16 +325,16 @@ public void TestReplicatorHeartbeatGetSet()
                 repl.Config.Heartbeat.Should().Be(TimeSpan.FromMinutes(5), "Because default Heartbeat Interval is 300 sec.");
             }
 
-            config.Heartbeat = TimeSpan.FromSeconds(60);
+            config = CreateConfig(true, false, false, null, null, null, null, null, null, null, -1, 300000, heartbeatInterval:60000);
```

Comment:
This will probably confuse anyone that looks at it.  So many arguments and most of them null!  I think readability would be better if this was not a function call (i.e. just make the object here if possible).

Reply:
cleaned up, and also applied to other similar cases